backend/Epicenter.Domain/Services/FaceService.cs

```diff
@@ -43,8 +43,10 @@ public async Task<List<RecognizedObject>> RecognizeAsync(string base64)
             result.ForEach(person =>
```

Comment:
this lambda expression is soo long, one of the reason they were introduced was the overhead of having to write a lot of code for SMALL functions, but this is not a small one :D define a delegate elsewhere and pass it instead of lambda